pyquil/quilatom.py

```diff
@@ -155,7 +163,23 @@ def register(cls, n: int) -> List["QubitPlaceholder"]:
         return [cls() for _ in range(n)]
 
 
-QubitDesignator = Union[Qubit, QubitPlaceholder, FormalArgument, int]
+QubitDesignator = Union[Qubit, QubitPlaceholder, FormalArgument, RSQubit, int]
+
+
+def convert_to_rs_qubit(qubit: QubitDesignator) -> RSQubit:
```

Comment:
I could see either way - maybe not a class since there's no state to store, but a `conversions` package could make sense. The way you have it here also makes sense to me, although I'd mark it private if you can.